### ISSUE TITLE
Pretrained rllib models?

### ISSUE THREAD

**Issue by leobxpan** [HumanCompatibleAI/overcooked-demo#25]
Thank you for the codebase. I'm trying to use a pretrained rllib agent to test the demo / environment but I didn't seem to find a proper one in the repo. Are there these model checkpoints available?

**Comment by micahcarroll** [comment]
Hi @leobxpan, there should now be one trained model that you can play with through the demo. The [readme file](https://github.com/HumanCompatibleAI/overcooked-demo/blob/master/server/static/assets/agents/RllibSelfPlay_CrampedRoom/README.md) should contain enough information to do the training yourself if you wish, and you can probably mostly port those hyperparams to other layouts too and get them to work with a little tweaking. We will be releasing other models in the next couple of weeks! Let us know if there are any issues!

**Comment by leobxpan** [comment]
Cool. Thank you for the work!

**Comment by mesutyang97** [comment]
Hi @leobxpan !

More checkpoints and the interface to evaluate agents natively have been formalized, and [made into a PR](https://github.com/HumanCompatibleAI/human_aware_rl/pull/11):.

Please keep an eye on it (I imagine it will be merged soon, pending review from my collaborators), but feel free to make recommendations if you wish to.

Thanks again for your interest in our work!

Best,
Mesut

**Comment by leobxpan** [comment]
Thanks for letting me know! @mesutyang97 Will def let you know if I get some good things to suggest :)

**Comment by mesutyang97** [comment]
sounds good

**Comment by upasana27** [comment]
I tried following this guide: https://github.com/HumanCompatibleAI/overcooked-demo/tree/master/server/static/assets/agents/RllibSelfPlay_CrampedRoom to add my pretrained rllib agent and play with it, however that didn't work. Trying to lead the already given example doesn't work either as shown in attached picture. Is there an ray compatibility issue, as I saw when the tutorial was pushed into this repository, the corresponding ray version in human aware rl repository was lower, is that the issue?
